libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.html

```diff
@@ -0,0 +1,50 @@
+<div id="map-section">
+  <tamu-gisc-drawer [width]="'20rem'" [position]="'left'">
+    <div class="content">
+      <h3>Map Controls</h3>
+      <input type="checkbox" name="county-state-toggle" (click)="stateToggle()">
+      <label for="county-state-toggle">Counties/State</label>
+      <br>
+      <input type="checkbox" name="infection-mortality-toggle" (click)="mortalityToggle()">
+      <label for="county-state-toggle">Infection Rate/Mortality Rate</label>
+      <input type="date" id="datepicker" value="2020-06-06" max="2020-06-06" min="2020-01-21">
```

Comment:
We also have `tamu-gisc-date-time-picker` but it hasn't been fully tested. So your call if you want to spend time trying that one out, definitely not necessary.

Reply:
I'm going to remove the button and the datepicker for now because there is no logic attached to them at the moment. when I add the date feature, I can look into the date-time-picker component.